geodns.go

```diff
@@ -69,6 +71,18 @@ func init() {
 func main() {
 	flag.Parse()
 
+	if *flaglogfile != "" {
```

Comment:
We shouldn't be logging that much, I think (and buffering might hide information before a panic).

buffering the query logging would be good though. :-)